event_routing_backends/processors/caliper/transformer_processor.py

```diff
@@ -10,6 +10,7 @@
 from event_routing_backends.processors.caliper.registry import CaliperTransformersRegistry
 from event_routing_backends.processors.mixins.base_transformer_processor import BaseTransformerProcessorMixin
 
+logger = getLogger(__name__)
 caliper_logger = getLogger('caliper_tracking')
```

Comment:
The changes in here and the xapi processor are just to keep the statement logging separate from debug output.